src/common/utility_unix.cpp

```diff
@@ -71,12 +72,24 @@ void setLaunchOnStartup_private(const QString &appName, const QString &guiName,
             qCWarning(lcUtility) << "Could not write auto start entry" << desktopFileLocation;
             return;
         }
+        // When running inside an AppImage, we need to set the path to the
+        // AppImage instead of the path to the executable
+        QString executablePath;
+        QString appImagePath = qEnvironmentVariable("APPIMAGE");
```

Comment:
This should be const, so I'd go for: `const auto appImagePath = qEnvironmentVariable("APPIMAGE");`